core/server/master/src/main/java/alluxio/master/block/BlockMasterClientServiceHandler.java

```diff
@@ -75,54 +73,30 @@ public GetServiceVersionTResponse getServiceVersion(GetServiceVersionTOptions op
   @Override
   public GetCapacityBytesTResponse getCapacityBytes(GetCapacityBytesTOptions options)
       throws AlluxioTException {
-    return RpcUtils.call(LOG, new RpcCallable<GetCapacityBytesTResponse>() {
-      @Override
-      public GetCapacityBytesTResponse call() throws AlluxioException {
-        return new GetCapacityBytesTResponse(mBlockMaster.getCapacityBytes());
-      }
-
-      @Override
-      public String toString() {
-        return String.format("getCapacityBytes: options=%s", options);
-      }
-    });
+    return RpcUtils.call(LOG, (RpcCallable<GetCapacityBytesTResponse>) () ->
+        new GetCapacityBytesTResponse(mBlockMaster.getCapacityBytes()),
+        "GetCapacityBytes", "GetCapacityBytes: options=%s", options);
   }
 
   @Override
   public GetUsedBytesTResponse getUsedBytes(GetUsedBytesTOptions options) throws AlluxioTException {
-    return RpcUtils.call(LOG, new RpcCallable<GetUsedBytesTResponse>() {
-      @Override
-      public GetUsedBytesTResponse call() throws AlluxioException {
-        return new GetUsedBytesTResponse(mBlockMaster.getUsedBytes());
-      }
-
-      @Override
-      public String toString() {
-        return String.format("getUsedBytes: options=%s", options);
-      }
-    });
+    return RpcUtils.call(LOG, (RpcCallable<GetUsedBytesTResponse>) () -> new GetUsedBytesTResponse(
+        mBlockMaster.getUsedBytes()), "GetUsedBytes", "GetUsedBytes: options=%s", options);
```

Comment:
it is kinda redundant to have the method name (e.g., "GetUsedBytes") to always prefix the description string.
Can we derive it inside `RpcUtils.call` from method name which is passed anyway?

Reply:
Updated, thanks.